platform/macos/app/Base.lproj/MainMenu.xib

```diff
@@ -128,13 +128,14 @@
                                 </connections>
                             </menuItem>
                             <menuItem isSeparatorItem="YES" id="aJh-i4-bef"/>
-                            <menuItem title="Page Setup…" keyEquivalent="P" id="qIS-W8-SiK">
-                                <modifierMask key="keyEquivalentModifierMask" shift="YES" command="YES"/>
+                            <menuItem title="Page Setup…" id="qIS-W8-SiK">
+                                <modifierMask key="keyEquivalentModifierMask" control="YES"/>
                                 <connections>
                                     <action selector="runPageLayout:" target="-1" id="Din-rz-gC5"/>
                                 </connections>
                             </menuItem>
-                            <menuItem title="Print…" keyEquivalent="p" id="aTl-1u-JFS">
+                            <menuItem title="Print…" id="aTl-1u-JFS">
+                                <modifierMask key="keyEquivalentModifierMask" control="YES"/>
```

Comment:
macosapp is the closest thing we have to an example for how to integrate the SDK into an application, so I’m hoping to [follow the HIG](https://developer.apple.com/library/prerelease/content/documentation/UserExperience/Conceptual/OSXHIGuidelines/InteractivityInput.html#//apple_ref/doc/uid/20000957-CH10-SW4) as much as possible:

> Respect the standard keyboard shortcuts and create app-specific shortcuts for frequently used commands.